Game/3d-game-project/2D/Scenes/House/cabin.gd

```diff
@@ -0,0 +1,20 @@
+extends Node2D
+class_name Cabin
+
+@onready var main = NodePath("/root/GameController")
+
+func _on_interactable_2d_focused(interactor: Interactor2D) -> void:
+	pass # Replace with function body.
```

Comment:
"Replace with function body" should be removed or the said function should be added